### ISSUE TITLE
built in protector

### ISSUE THREAD

**Issue by cornernote** [cornernote/minetest-skyblock#36]
protector should protect the initial island

it may grow as the player progresses, or we may still rely on the protector mod's node to do this

**Comment by cornernote** [comment]
protector should be optional in depends.txt

**Comment by cornernote** [comment]
also, protectors can be used to grief... people can place them in other people's areas

**Comment by HybridDog** [comment]
>protector should be optional in depends.txt

if the protector node doesn't exist, you could give a steel block instead as reward

**Comment by cornernote** [comment]
> if the protector node doesn't exist, you could give a steel block instead as reward

problem is i dont want them to have steel until level2... any other ideas as an alt reward for protector?

**Comment by HybridDog** [comment]
maybe 10 stone nodes

**Comment by Sokomine** [comment]
At least one locked chest?

**Comment by HybridDog** [comment]
How can anyone get a cactus?

**Comment by doyousketch2** [comment]
I think whatever protection scheme you use, it should automatically cover from about 10 blocks down about 50 blocks up, so only the owner of the island can do stuff in that column of vertical space.

**Comment by cornernote** [comment]
yeah, i have seen someone has gone around owning a lot of sky by running a
single line of dirt along with a protector every 20 blocks.  they must have
spent a lot of time on it cos there it about 20-50 protector nodes along
their paths.  it means others have a hard time building up because they are
blocked off

**Comment by HybridDog** [comment]
You could make protection only work at least 30 nodes above the player bases that people begin to build their houses in different heights and don't stay at their given places.
